src/enforcer.js

```diff
@@ -258,22 +322,24 @@ export class TrustedTypesEnforcer {
     const that = this;
 
     ['appendChild', 'insertBefore', 'replaceChild'].forEach((fnName) => {
-      this.wrapFunction_(
-          Node.prototype,
-          fnName,
-          /**
-         * @this {Node}
-         * @param {function(!Function, ...*)} originalFn
-         * @return {*}
-         */
-          function(originalFn, ...args) {
-            return that.enforceTypeInScriptNodes_
-                .bind(that, this, /* checkParent */ false, originalFn)
-                .apply(that, args);
-          });
+      if (this.windowObject.Node) {
+        this.wrapFunction_(
+            this.windowObject.Node.prototype,
+            fnName,
+            /**
+           * @this {Node}
+           * @param {function(!Function, ...*)} originalFn
+           * @return {*}
+           */
+            function(originalFn, ...args) {
+              return that.enforceTypeInScriptNodes_
+                  .bind(that, this, /* checkParent */ false, originalFn)
+                  .apply(that, args);
+            });
+      }
     });
     this.wrapFunction_(
-        insertAdjacentObject,
+        this.insertAdjacentObjectPrototype,
```

Comment:
This might be null, should we not skip wrapFunction_ in this case, like above? Also in uninstall().

Reply:
I verify truthiness in the wrap/restore functions.

Reply:
Done.